### ISSUE TITLE
How to put an exe to startup folder?

### ISSUE THREAD

**Issue by fancyfanta** [Squirrel/Squirrel.Windows#939]
I searched a lot but couldn't find info about it. How can I put an exe's shortcut to startup folder with Squirrel Setup?

**Comment by Mierenga** [comment]
I'm not sure if this is available in Squirrel (I haven't seen it), but code for this feature can be pretty simple. Check out this Gist: [StartupShortcut.cs](https://gist.github.com/Mierenga/a8a1fcdbee5555ac9249aab7837da0bd)
Works well when used in conjunction with SquirrelAware event handling.

**Comment by Thieum** [comment]
@ErcinDedeoglu did the solution provided by @Mierenga fixed your issue? If so, you can close this.

**Comment by ErcinDedeoglu** [comment]
No, there is no solution right now. We need documentation for it. @Thieum
